importer/DkBygningDa.py

```diff
@@ -14,15 +14,25 @@ def set_location(self):
         """
         Use self.municipality because IT'S PLACE NOT MUNICIPALITY
```

Comment:
A general comment on docstrings:
The first line of a docstring should end in a full stop, be in imperative mood (as if you're instructing someone), and fit on a single line.

Subsequent lines are separated from the first by a single new line.

If the docstring consists of only one line it should fit on the same line as the leading/ending  `"""`